visallo/visallo
# modified ontology selector to return user visible concepts and properties if showHiddenConcepts or showHiddenProperties is set to true

Add the ability to send a flag so that the component will render properties and concepts set to `userVisible=false`


Testing Instructions: 
- Make sure when adding new entities, concepts that are marked `userVisible=false` does not show up in list
- Make sure when adding new properties, properties that are marked `userVisible=false` does not show up in list

Points of Regression: N/A

NO CHANGELOG
